packages/kernel/src/kernel.ts

```diff
@@ -512,6 +515,25 @@ export abstract class BaseKernel implements IKernel {
     this._sendMessage(message);
   }
 
+  /**
+   * Handle an is_complete_request message
+   *
+   * @param msg The parent message.
+   */
+  private async _is_complete_request(msg: KernelMessage.IMessage): Promise<void> {
```

Comment:
Minor nit-pick: since this part is in TypeScript we could follow the rest of the code use camel casing for the name of the method:

```suggestion
  private async _isCompleteRequest(msg: KernelMessage.IMessage): Promise<void> {
```

Reply:
just fixed